eng/Versions.props

```diff
@@ -20,7 +20,6 @@
   <PropertyGroup>
     <UsingToolIbcOptimization>true</UsingToolIbcOptimization>
     <UsingToolVisualStudioIbcTraining>true</UsingToolVisualStudioIbcTraining>
-    <UsingToolMicrosoftNetCompilers>true</UsingToolMicrosoftNetCompilers>
```

Comment:
Consider also removing `<MicrosoftNetCompilersToolsetVersion>4.14.0-3.25218.8</MicrosoftNetCompilersToolsetVersion>` from this file

Reply:
We cannot do this--we actually still depend on this package for tests (in the bootstrap environment) even when we don't use it to provide the compiler for our assemblies.